app/ui/lib/Message.tsx

```diff
@@ -73,18 +76,24 @@ export const Message = ({
   variant = 'success',
   cta,
   icon,
+  hideableKey,
 }: MessageProps) => {
+  const [hidden, setHidden] = useState(false)
+  if (hidden || (hideableKey && hideableKey in localStorage)) {
```

Comment:
I think we should nest these all under one key maybe? makes them much easier to nuke and stuff